libraries/Gamebuino-Meta/src/utility/Sound/Sound.cpp

```diff
@@ -262,6 +287,10 @@ void Sound::update() {
 			handlers[i] = 0;
 		}
 	}
+
+	if (fx_channels[0].handler) {
```

Comment:
if you have a configurable amount of fx_channels, you should iterate over them. However, I do not see a case where you would need multiple, so why make it an array at all?